crypto/provider_conf.c

```diff
@@ -272,8 +272,8 @@ static int provider_conf_activate(OSSL_LIB_CTX *libctx, const char *name,
     return ok;
 }
 
-static int provider_conf_parse_bool_setting(const char *confname,
-                                            const char *confvalue, int *val)
+int ossl_provider_conf_parse_bool_setting(const char *confname,
+                                          const char *confvalue, int *val)
```

Comment:
I could not see why this change is needed?

Reply:
I'll need this to actually make use of any boolean parameters that are added.